commit e462f9667a6e28de948d0341768167c5d96c0f7e
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Sun Oct 7 17:37:54 2012 +0300

    Fix the build without KInotify again.
    
    CCMAIL: dvratil@redhat.com

diff --git a/services/filewatch/nepomukfilewatch.cpp b/services/filewatch/nepomukfilewatch.cpp
index 83045da..9ccf7b0 100644
--- a/services/filewatch/nepomukfilewatch.cpp
+++ b/services/filewatch/nepomukfilewatch.cpp
@@ -114,7 +114,9 @@ namespace {
 
 Nepomuk2::FileWatch::FileWatch( QObject* parent, const QList<QVariant>& )
     : Service( parent )
+#ifdef BUILD_KINOTIFY
     , m_dirWatch( 0 )
+#endif
 {
     // Create the configuration instance singleton (for thread-safety)
     // ==============================================================
